src/daemon/dlt_daemon_common.c

```diff
@@ -318,6 +386,12 @@ int dlt_daemon_init(DltDaemon *daemon,
 
     daemon->state = DLT_DAEMON_STATE_INIT; /* initial logging state */
 
+#ifdef DLT_TRACE_LOAD_CTRL_ENABLE
+    daemon->preconfigured_trace_load_settings = NULL;
```

Comment:
Do we need init value for  daemon->preconfigured_trace_load_settings_count?

Reply:
Yes, it is good practice to initialize memory, so Nullchecks work